### PR TITLE
[IMP] * : renaming "Send SMS text message" into "Send SMS"

### PR DESCRIPTION
**Pull request by malb-odoo**
This commit rename all occurrence of the "send SMS text message" to "Send SMS"

task: 4213146

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing_sms/views/mailing_list_views.xml
Comment (epictete) [inline addons/mass_mailing_sms/views/mailing_list_views.xml:1]: Not sure it's linked to the task (i.e. the action)?

File: addons/sms/data/iap_service_data.xml
Comment (epictete) [inline addons/sms/data/iap_service_data.xml:1]: Same here?

**Review by epictete — COMMENTED** [review]
Same here, could you make your commit title a bit more explicit that just `renaming`?

File: addons/mass_mailing_sms/views/mailing_list_views.xml
Comment (malb-odoo) [reply addons/mass_mailing_sms/views/mailing_list_views.xml:1]: i prefer to be consistent and rename every occurrence, anyway the "text message" is useless for me ahah

**Review by epictete — APPROVED** [review]
@robodoo r+

**Comment by epictete** [comment]
@robodoo r+

**Pull request closed without merging**